src/themes/dark.scss

```diff
@@ -3,13 +3,43 @@
   --wxc-text-color: #{$wxc-white-alpha-100};
   --wxc-meeting-border: 1px solid #{$wxc-white-alpha-30};
   --wxc-meeting-box-shadow: none;
-  
+
   --wxc-icon--color: #{$wxc-white-alpha-100};
   --wxc-icon--active-color: #{$wxc-green-50};
   --wxc-icon--muted-color: #{$wxc-red-50};
 
   .wxc-old-icon { display: none; }
   .wxc-icon { display: block; }
+  --wxc-button--cancel--normal--bg: #{$wxc-red-60};
```

Comment:
put these together with the other vars (above the 2 classes above)

and sort them by: button type (already done), button state, css property. Eg: 
```css
--wxc-button--cancel--normal--color: #{$wxc-white-alpha-95};
--wxc-button--cancel--normal--border: none;
```

and you can leave a blank line between button states for better readability